courses/free-courses-ko.md

```diff
@@ -153,12 +155,12 @@
 * [AngularJS](https://www.youtube.com/playlist?list=PLs_XsVQJKaBk_JN5RctLmmVrGwEzpzqaj) - 양재동 코드랩 (YouTube)
 
 
-#### Node.js
+#### <a id="node.js"></a>Node.js
 
 * [모던 웹을 위한 Node.js 프로그래밍](https://www.youtube.com/playlist?list=PL9kzervdzKxy-c7UQgZbnZIr14DCPM87j)
 
 
-#### React.js
+#### <a id="react.js"></a>React.js
```

Comment:
Change the index link to `#reactjs` instead

Reply:
Ok

Reply:
Reverted in f7896fab0f35ec244ebb5b7168f93e163b6745fc